### ISSUE TITLE
Indefined variable right

### ISSUE THREAD

**Issue by mvestrotech** [pluginsGLPI/mreporting#160]
A problem encountered when creating a profile to configure the right Mreporting on the profile in simplified.

![image](https://user-images.githubusercontent.com/23355435/42811586-43d1ef30-89bb-11e8-9a76-dd2ee1970cc2.png)

glpi : 9.2.4
Mreporting : 1.4.1

**Comment by amma35** [comment]
Same problem, SQL error during update: 
```
  *** MySQL query error:
  SQL: ALTER IGNORE TABLE glpi_plugin_mreporting_profiles
                ADD UNIQUE INDEX `profiles_id_reports` (`profiles_id`, `reports`)
  Error: Erreur de syntaxe près de 'IGNORE TABLE glpi_plugin_mreporting_profiles
                ADD UNIQUE INDEX `p' à la ligne 1
  Backtrace :
  plugins\mreporting\hook.php:162                    
  inc\plugin.class.php:731                           plugin_mreporting_install()
  front\plugin.form.php:48                           Plugin->install()
```

**Comment by cedric-anne** [comment]
Fixed by #168 and #169 .
